disperser/dataapi/operator_handler.go

```diff
@@ -84,73 +84,90 @@ func NewOperatorHandler(logger logging.Logger, metrics *Metrics, chainReader cor
 	if err != nil {
 		oh.logger.Warn("failed to fetch operator info", "operatorId", operatorId, "error", err)
 		return &OperatorPortCheckResponse{}, err
 	}
 
 	operatorSocket := core.OperatorSocket(operatorInfo.Socket)
-	v1RetrievalSocket := operatorSocket.GetV1RetrievalSocket()
-	v1RetrievalPortOpen := checkIsOperatorPortOpen(v1RetrievalSocket, 3, oh.logger)
-	v1RetrievalOnline, v1RetrievalStatus := false, "port closed or unreachable"
-	if v1RetrievalPortOpen {
-		v1RetrievalOnline, v1RetrievalStatus = checkServiceOnline(ctx, "node.Retrieval", v1RetrievalSocket, 3*time.Second)
-	}
-
-	v1DispersalSocket := operatorSocket.GetV1DispersalSocket()
-	v1DispersalPortOpen := checkIsOperatorPortOpen(v1DispersalSocket, 3, oh.logger)
-	v1DispersalOnline, v1DispersalStatus := false, "port closed or unreachable"
-	if v1DispersalPortOpen {
-		v1DispersalOnline, v1DispersalStatus = checkServiceOnline(ctx, "node.Dispersal", v1DispersalSocket, 3*time.Second)
-	}
 
-	v2DispersalOnline, v2DispersalStatus := false, ""
-	v2DispersalSocket := operatorSocket.GetV2DispersalSocket()
-	if v2DispersalSocket == "" {
-		v2DispersalStatus = "v2 dispersal port is not registered"
+	retrievalOnline, retrievalStatus := false, "v2 retrieval port closed or unreachable"
+	retrievalSocket := operatorSocket.GetV2RetrievalSocket()
+	if retrievalSocket == "" {
+		retrievalStatus = "v2 retrieval port is not registered"
 	} else {
-		v2DispersalPortOpen := checkIsOperatorPortOpen(v2DispersalSocket, 3, oh.logger)
-		if !v2DispersalPortOpen {
-			v2DispersalStatus = "port closed or unreachable"
-		} else {
-			v2DispersalOnline, v2DispersalStatus = checkServiceOnline(ctx, "node.v2.Dispersal", v2DispersalSocket, 3*time.Second)
+		retrievalPortOpen := checkIsOperatorPortOpen(retrievalSocket, 3, oh.logger)
+		if retrievalPortOpen {
+			retrievalOnline, retrievalStatus = checkServiceOnline(ctx, "validator.Retrieval", retrievalSocket, 3*time.Second)
 		}
 	}
 
-	v2RetrievalOnline, v2RetrievalStatus := false, ""
-	v2RetrievalSocket := operatorSocket.GetV2RetrievalSocket()
-	if v2RetrievalSocket == "" {
-		v2RetrievalStatus = "v2 retrieval port is not registered"
+	dispersalOnline, dispersalStatus := false, "v2 dispersal port closed or unreachable"
+	dispersalSocket := operatorSocket.GetV2DispersalSocket()
+	if dispersalSocket == "" {
+		dispersalStatus = "v2 dispersal port is not registered"
 	} else {
-		v2RetrievalPortOpen := checkIsOperatorPortOpen(v2RetrievalSocket, 3, oh.logger)
-		if !v2RetrievalPortOpen {
-			v2RetrievalStatus = "port closed or unreachable"
-		} else {
-			v2RetrievalOnline, v2RetrievalStatus = checkServiceOnline(ctx, "node.v2.Retrieval", v2RetrievalSocket, 3*time.Second)
+		dispersalPortOpen := checkIsOperatorPortOpen(dispersalSocket, 3, oh.logger)
+		if dispersalPortOpen {
+			dispersalOnline, dispersalStatus = checkServiceOnline(ctx, "validator.Dispersal", dispersalSocket, 3*time.Second)
```

Comment:
I like removing the version from the service name. eventually we should move versioning to package levels to better support compat issues.

Reply:
The renaming was done as part of documentation overhaul in https://github.com/Layr-Labs/eigenda/pull/1125